v2.7.0 Windows Installation Error: symlink

We currently see the following Windows installation error on conda-forge:
```
-- Installing: D:/bld/adios2_1610478470869/_h_env/Library/bin/adios2_reorganize_wrapper
CMake Error at source/utils/cmake_install.cmake:52 (file):
  file INSTALL cannot read symlink
  "D:/bld/adios2_1610478470869/work/source/utils/adios_reorganize/bp2h5" to
  duplicate at "D:/bld/adios2_1610478470869/_h_env/Library/bin/bp2h5": No
  such file or directory.
Call Stack (most recent call first):
  source/cmake_install.cmake:38 (include)
  cmake_install.cmake:43 (include)


NMAKE : fatal error U1077: 'D:\bld\adios2_1610478470869\_build_env\Library\bin\cmake.exe' : return code '0x1'
Stop.
Traceback (most recent call last):
```

Issue: https://github.com/conda-forge/adios2-feedstock/pull/24
Install logic: https://github.com/conda-forge/adios2-feedstock/blob/5a2ecdeebeac8bc9eb7594661701d95db63c63af/recipe/bld.bat

--- a/source/utils/CMakeLists.txt
+++ b/source/utils/CMakeLists.txt
@@ -84,17 +84,14 @@ configure_file(
   ${PROJECT_BINARY_DIR}/adios2_reorganize_wrapper
   @ONLY
 )
+
 if(ADIOS2_HAVE_HDF5)
   install(PROGRAMS ${PROJECT_BINARY_DIR}/adios2_reorganize_wrapper
-    RENAME adios2_reorganize_wrapper${ADIOS2_EXECUTABLE_SUFFIX}
-    DESTINATION ${CMAKE_INSTALL_BINDIR}
-  ) 
-  install(PROGRAMS adios_reorganize/bp2h5
     RENAME bp2h5${ADIOS2_EXECUTABLE_SUFFIX}
     DESTINATION ${CMAKE_INSTALL_BINDIR}
   )
   if(ADIOS2_HAVE_MPI)
-    install(PROGRAMS adios_reorganize/bp2h5_mpi
+    install(PROGRAMS ${PROJECT_BINARY_DIR}/adios2_reorganize_wrapper
       RENAME bp2h5_mpi${ADIOS2_EXECUTABLE_SUFFIX}
       DESTINATION ${CMAKE_INSTALL_BINDIR}
     )

--- a/source/utils/adios_reorganize/bp2h5
+++ b/source/utils/adios_reorganize/bp2h5
@@ -1,1 +1,0 @@
-adios2_reorganize_wrapper

--- a/source/utils/adios_reorganize/bp2h5_mpi
+++ b/source/utils/adios_reorganize/bp2h5_mpi
@@ -1,1 +1,0 @@
-adios2_reorganize_wrapper

